Ownership/Non-UK/Shannon.txt

```diff
@@ -80,57 +80,57 @@ OWNER:SNB:SNE
 BORDER:4DS/4DU-1:4DS/4DU-2:4DS/4DU-3:4DS/4DU-4:4DS/4DU-5:4DS/4DU-6
 
 SECTOR:EISN 4DS:35500:66000
-OWNER:SNB:SNE
+OWNER:SN8:SNB:SNE
 BORDER:4DS/4DU-1:4DS/4DU-2:4DS/4DU-3:4DS/4DU-4:4DS/4DU-5:4DS/4DU-6
 
 SECTOR:EISN 4GU:24500:35500
 OWNER:SNB:SNE
 BORDER:4GS/4GU-1:4GS/4GU-2:4GS/4GU-3:4GS/4GU-4:4GS/4GU-5
 
 SECTOR:EISN 4GS:35500:66000
-OWNER:SNB:SNE
+OWNER:SN8:SNB:SNE
 BORDER:4GS/4GU-1:4GS/4GU-2:4GS/4GU-3:4GS/4GU-4:4GS/4GU-5
 
 ;SOTA
 SECTOR:Shannon SOTA:0:0
 OWNER:SNS:SNO:SNW:SNE
```

Comment:
@Paulmc98 should this include SNB too presumably? We use this for COPX fallback
```suggestion
OWNER:SNS:SNO:SNW:SNB:SNE
```

Reply:
The line should read "OWNER:SNS:SNW:SNB:SNE". You are correct with the addition of SNB but SNO can be removed as OSCAR will never control SOTA

Reply:
Edited my suggestion!